src/state/machines/expansion/overfishing.ts

```diff
@@ -60,8 +61,18 @@ export const state: {
 } = {
   [cardPrefix]: {
     tags: ["policy", cardPrefix],
-    initial: "discardMarketFish",
+    initial: "checkingProtection",
     states: {
+      checkingProtection: {
+        always: [
+          {
+            target: `#turn.${StrictProtection.cardPrefix}.stageProtection`,
+            actions: `${cardPrefix}Done`,
+            guard: { type: "isPolicyCardActive", params: StrictProtection.cardName },
+          },
+          { target: "discardMarketFish" },
+        ],
```

Comment:
Cool. That is neat!

Maybe we should have a `protectedStage` with oil/overshishing/... as an `effect` and  `protection` card as an cause? With text like "Protection measures saved ecosfera..."

Reply:
I updated the text, but adding the targeted card to the stage proved to be a bit more difficult because we completely remove the card from the game after it has been shown on the stage. I will add a new issue with the comments you made about the "exhausted" cards on my previous PR and also indicate there that it should be used to improve the protection stage👍🏻